### PR TITLE
chore: circleci

### PR DESCRIPTION
**Pull request by pheel**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /-bf\./
+          context: org-global-v2
+      - docker/push_tags:
+          name: push-image-tags
+          requires:
+            - build-image
```
Comment (GuillaumeTech) [inline .circleci/config.yml:42]: I think it requires a repo parameter to push to ? https://circleci.com/developer/orbs/orb/dialogue/docker#commands-push_tags
or is passed another way ?

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+          context: org-global-v2
+      - python/pylama:
+          name: pylama
+          context: org-global-v2
+      - python/test:
+          name: test
+          tests_directory: "tests/core/botfront tests/addons"
+          pytest_options: --no-cov
+          context: org-global-v2
+      - docker/build_image:
```
Comment (GuillaumeTech) [inline .circleci/config.yml:26]: would this need a dockerfile parameters so it references our dockerfile ? 
https://circleci.com/developer/orbs/orb/dialogue/docker#commands-build_image

**Review by GuillaumeTech — COMMENTED** [review]
Few comments, is there a way to see the result of a run on circle ci somewhere ?

File: .circleci/config.yml (hunk 1 above)
Comment (pheel) [reply .circleci/config.yml:42]: Just using the default?

File: .circleci/config.yml (hunk 2 above)
Comment (pheel) [reply .circleci/config.yml:26]: You're right, since our Dockerfile is not the default.

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+          dockerfile: docker/Dockerfile_botfront
+          filters:
+            branches:
+              only: master
+            tags:
+              only: /-bf\./
+          context: org-global-v2
+      - docker/push_tags:
+          name: push-image-tags
+          requires:
```
Comment (pheel) [inline .circleci/config.yml:42]: also need to change registry to docker hub

Comment (pheel) [reply .circleci/config.yml:42]: @mbelang Could I use this pipeline to push to the Dialogue org @ Docker Hub? Or should we instead somehow use our old Docker credentials?

**Pull request closed without merging**